hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/pbj/PbjConverter.java

```diff
@@ -1309,6 +1310,21 @@ public static <T extends Record, R extends GeneratedMessageV3> R pbjToProto(
         }
     }
 
+    public static @NonNull Optional<SchedulableTransactionBody> fromPbj(
```

Comment:
Should be named toPbj() according to others' name conversion.

Reply:
Fixed.